cl/api/management/commands/cl_retry_webhooks.py

```diff
@@ -51,6 +53,29 @@ def retry_webhook_events() -> int:
     return len(webhook_events_to_retry)
 
 
+def delete_old_webhook_events() -> int | None:
+    """Delete webhook events older than DAYS_TO_DELETE days.
+    This is executed once a day at 12:00 UTC (4:00 PDT)
+
+    :return: The number of deleted webhooks events or None if it's not time to
+    execute the method.
+    """
+
+    minute_of_the_day = now().hour * 60 + now().minute
+    # 12:00 UTC -> 4:00 PDT (minute 720 of the day)
+    if minute_of_the_day == 720:
```

Comment:
I know I kind of got you thinking this way, but wouldn't it be more clear to do something like:

```
if now.hour == 0 and now.minute == 0:
```

Or:

```
if now().time == time(0, 0):
```

I have no idea how to do that properly, but I'm guessing there's a better way to check if it's a specific time than counting the minutes in the day. I was just doing minutes in day because I wanted to do a modulo trick, but it looks silly now, right?

Reply:
Yeah, this is clearer, it worked the option: ` if now.hour == 0 and now.minute == 0:`

Reply:
Wow, I really thought that'd be wrong somehow!